swupd/rename.go

```diff
@@ -93,6 +93,7 @@ func linkRenamePair(renameTo, renameFrom *File) {
 	renameTo.Rename = true
 	renameFrom.Rename = true
 	renameFrom.Type = TypeUnset
+	renameFrom.Hash = renameTo.Hash
```

Comment:
This should be gone when we rebase with the fix/test by Matthew.

Reply:
done and in master